react/README.md

```diff
@@ -0,0 +1,158 @@
+# Introduction
+
+This React sample demonstrates how to perform login actions that will initiate flow endpoint operations are used only to implement custom authentication UIs. 
+OIDC/OAuth 2 requests initiate the flow and redirect the browser to the custom authentication UI (which is configured in the application through the application’s `loginPageUrl` property.)
+
+This example contains code for SPA that( by default ) calls OAuth 2.0 implicit flow, where the access token is returned immediately without an extra authorization code exchange step.
+If succeeded, the  OAuth 2.0 protected resource ( [UserInfo Endpoint](https://apidocs.pingidentity.com/pingone/customer/v1/api/auth/p1-a_Authorize/#UserInfo-endpoint) ) that returns claims about the authenticated end user can be tested by clicking __"Show user information"__ link.
```

Comment:
Suggestion for rewording this sentence:

If successful, the OAuth 2.0 protected resource (...) returns claims about the authenticated end user and can be test by clicking the __"Show user information"__ link.